sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OrchestrationController.java

```diff
@@ -166,8 +189,10 @@ Object llamaGuardInputFiltering(
     final OrchestrationChatResponse response;
     try {
       response = service.llamaGuardInputFilter(enabled);
-    } catch (OrchestrationClientException e) {
-      final var msg = "Failed to obtain a response as the content was flagged by input filter.";
+    } catch (OrchestrationInputFilterException e) {
+      final var msg =
+          "Failed to obtain a response as the content was flagged by input filter. Error %d"
+              .formatted(e.getStatusCode());
```

Comment:
```suggestion
      var msg =
          "Failed to obtain a response as the content was flagged by input filter. Error %d"
              .formatted(e.getStatusCode());
      if(e.getLlamaGuard38b() != null){
        msg += " Violent crimes are %s".formatted(e.getLlamaGuard38b().isViolentCrimes());
      }
```
Made is consistent with the azure filter